src/Inputs/MaskedInput.tsx

```diff
@@ -1,14 +1,17 @@
 import React, { useState, useEffect } from 'react';
-import { LabelLayout, LabelLayoutProps, InputFragment } from '../Fragments';
+import {
+    LabelLayout,
+    LabelLayoutProps,
+    InputFragment,
+    InputFragmentProps,
+} from '../Fragments';
 
 export enum MaskedInputPreset {
     DOLLAR = 'DOLLAR',
     PERCENTAGE = 'PERCENTAGE',
 }
 
-export interface MaskedInputProps
-    extends LabelLayoutProps,
-        React.InputHTMLAttributes<HTMLInputElement> {
+export interface MaskedInputProps extends LabelLayoutProps, InputFragmentProps {
```

Comment:
No need for HTML? There is a props spread, is that covered by the others?

Reply:
InputFragmentProps already extends HTMLInputElement.